setup.py

```diff
@@ -830,6 +801,9 @@ def objects_to_spec(preset_name: str,
         )
     )
 
+    # Collect previous fork's builders starting with fork, e.g. [bellatrix, altair, phase0]
+    builders = [spec_builders[fork] for fork in collect_prev_forks(fork)]
```

Comment:
Maybe reverse the order so the built `minimal.py` & `mainnet.py` will start with phase0 hardcoded code.

```suggestion
    # Collect builders with the reversed previous forks
    # e.g. `[bellatrix, altair, phase0]` -> `[phase0, altair, bellatrix]`
    builders = [spec_builders[fork] for fork in collect_prev_forks(fork)[::-1]]
```